### DISCUSSION TITLE
Different folder within OneDrive

### DISCUSSION THREAD

**Discussion by Jakewbb** [lavinir/hassio-onedrive-backup#159] [Ideas]
It would be great if we could get it to talk to a folder in OneDrive outside the "Apps" folder.

**Comment by Jakewbb** [comment]
e.g.  Onedrive>Backups>HASS

**Comment by lavinir** [comment]
This is not possible without granting the addon full rw access to all of your OneDrive.
https://github.com/lavinir/hassio-onedrive-backup?tab=readme-ov-file#backup-location-in-onedrive
